src/main/java/fence/ui/Ui.java

```diff
@@ -0,0 +1,66 @@
+package fence.ui;
```

Comment:
The UI responses in the Ui class are user-friendly and straightforward, goodjob!